src/components/addApplications.tsx

```diff
@@ -0,0 +1,39 @@
+"use client";
+
+import { database } from "@/app/firebaseConfig";
+import ApplicationForm from "@/components/application-form";
+import Typography from "@/components/Typography/typography";
+import { ref, set, push } from "@firebase/database";
+import { toast } from "sonner";
+
+export default function AddApplications() {
+  const handleSubmit = async (application: {
+    startup_name: string;
+    full_name: string;
+    email: string;
+    phone: string;
+    city: string;
+    startup_description: string;
+    startup_stage: string;
+  }) => {
+    try {
+      const applicationsRef = ref(database, "applications");
```

Comment:
Could you create a service please 
an applicationsService.ts

and from locations you should just call the service
applicationsService.get (or, update, or delete) 

if you dont do it we will have a lot of repited code on the project